src/common/m_derived_types.fpp

```diff
@@ -311,9 +311,10 @@ module m_derived_types
     type chemistry_parameters
         character(LEN=name_len) :: cantera_file !< Path to Cantera file
 
-        logical :: advection
         logical :: diffusion
         logical :: reactions
+
+        integer :: gamma_method
```

Comment:
can we add a comment here that indicates what `gamma_method` ultimately means?

Reply:
Done.